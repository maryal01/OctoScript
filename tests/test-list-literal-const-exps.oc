list<int> li = [1, 2, -3, 1 * 3, 5 * 1, 8 / 9, 2 % 1];
list<float> lf = [-1.4, 3.6*72.624, 9.8 / 2.8, 852.95 + 95.82];
list<bool> lb = [true, false, true & false, false | true];

tuple<int, int, int, bool, bool, float> t = (1, -62, 27*543, false & true, !false, 48.2 * -38.8);

print(string_of_list(li));
print(string_of_list(lf));
print(string_of_list(lb));
print(string_of_tuple(t));