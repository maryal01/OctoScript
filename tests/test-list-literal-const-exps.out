[1, 2, -3, 3, 5, 0, 0][-1.400000, 261.446400, 3.500000, 948.770000][true, false, false, true](1, -62, 14661, false, true, -1870.160000)